src/parser/token.rs

```diff
@@ -1,8 +1,11 @@
+#[derive(Debug, PartialEq)]
 pub enum Token {
   LPar,        // (
   RPar,        // )
 
   LInt(u64),
 
   OpPlus,      // +
+
+  EOF          // End of file
```

Comment:
`Eof`

Reply:
Also, `LInt` should be `i64`.

Reply:
More complexity to your lexer, yes!!! Now integers may start with a leading `-`. >:D